src/main/java/com/github/princesslana/slothbot/Moderator.java

```diff
@@ -0,0 +1,85 @@
+package com.github.princesslana.slothbot;
+
+import com.eclipsesource.json.Json;
+import com.eclipsesource.json.JsonArray;
+import com.eclipsesource.json.WriterConfig;
+import com.google.common.base.Charsets;
+import com.google.common.collect.HashMultimap;
+import com.google.common.io.MoreFiles;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.Set;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class Moderator {
+  private static final Logger LOG = LogManager.getLogger(Limiter.class);
+
+  private final HashMultimap<String, String> guildToModeratorRoles = HashMultimap.create();
+  private final Path savePath;
+
+  public Moderator(Path savePath) {
+    this.savePath = savePath;
+  }
+
+  public void add(String guildId, String roleId) {
+    if (!guildToModeratorRoles.containsEntry(guildId, roleId)) {
```

Comment:
HashMultimap doesn't allow duplicates, so we can just put.

Reply:
I've never used it before, I got a bit weirded out by this section in the docs for remove `If multiple key-value pairs in the multimap fit this description, which one is removed is unspecified.` but I might've been reading too much into that